src/VisualStudio/IntegrationTest/TestUtilities/Input/SendKeys.cs

```diff
@@ -193,11 +83,6 @@ private void SendInputs(NativeMethods.INPUT[] inputs)
                 {
                     IntegrationHelper.SetForegroundWindow(foregroundWindow);
                 }
-
-                if (inputBlocked)
-                {
-                    IntegrationHelper.UnblockInput();
```

Comment:
Any guess if this was causing problems for us?

Reply:
Nothing conclusive. I eliminated it simply so the integration test runs are a closer match to the state when devs run the tests.